llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -2069,7 +2091,7 @@ SDValue NVPTXTargetLowering::LowerBITCAST(SDValue Op, SelectionDAG &DAG) const {
 SDValue NVPTXTargetLowering::LowerBUILD_VECTOR(SDValue Op,
                                                SelectionDAG &DAG) const {
   EVT VT = Op->getValueType(0);
-  if (!(Isv2x16VT(VT) || VT == MVT::v4i8))
+  if (!isPackedVectorTy(VT) || !VT.is32BitVector())
```

Comment:
Nit: Rephrased as `if (!(isPackedVectorTy(VT) && VT.is32BitVector()))` would be easier to read, IMO.